javascript/compute-height.js

```diff
@@ -1,25 +1,14 @@
-const TAB_OFFSET_PADDING = 5;
-
-function getRecursiveParentNode(el, n) {
-    if(n <= 0) return el;
-    if(el === null || el.parentNode === null) return el;
-    return getRecursiveParentNode(el.parentNode, n-1);
-}
-
-const getDynamicElementFromContainer = (container) => {
-    const webuiParentDepth = 7;
-    return getRecursiveParentNode(container, webuiParentDepth);
-}
-
 function computeComfyuiElementHeight() {
     const tab = document.getElementById("tab_comfyui_webui_root");
     const container = document.getElementById("comfyui_webui_container");
     const footerToRemove = document.querySelector('#footer');
-    const dynamicElement = getDynamicElementFromContainer(container);
+    const tabs = document.getElementById("tabs") ?? null;
+    const tab_nav = tabs ? tabs.querySelector(".tab-nav") : null;
+    console.log(tab_nav)
 
-    if(dynamicElement !== null) {
-        const height = dynamicElement.offsetHeight;
-        container.style.height = `calc(100% - ${height-TAB_OFFSET_PADDING}px)`;
+    if(tab_nav !== null) {

```

Comment:
Do you think
```js
if(tab_nav) {
```
would be better?

Reply:
In js `if(reference)` is the same as `!= undefined && != null` or `!== undefined && !== null`? or something else? Just to be safe I went the explicit route.